duckduckgo/Android
# Add support for opening "target blank" urls in new tabs

**Task/Issue URL**:  https://app.asana.com/0/361428290920652/1134893348703258

**Description**:
Adds support for opening links with `target="_blank"` in a new tab and fixes https://github.com/duckduckgo/Android/issues/502. Completes work started in https://github.com/duckduckgo/Android/pull/511 which added support for opening links in new tabs. In this PR we do not open additional tabs and pass along the transport payload `WebView.WebViewTransport` to the new tab.

Note: this PR is being merged into `feature/mia/support_target_blank` along with https://github.com/duckduckgo/Android/pull/511. The consolidated `feature/mia/support_target_blank` branch will then be merged (not squashed) into develop to preserve contributor history.

**Steps to test this PR**:

**TEST ONE: Test js window.open button**
1. Visit https://www.w3schools.com/code/tryit.asp?filename=G8SKXWB6508R
1. Press "Run"
1. Tap the "click on me" button
1. Ensure a new url is opened in a new tab and note that the tab count is incremented and animates.
1. Check that the original tab is unchanged and still on the w3cschools website.

**TEST TWO: Test href**
1. Visit https://www.w3schools.com/code/tryit.asp?filename=G8SKXWB6508R
1. Press "Run"
1. Tap the "click on me" link
1. Ensure a new url is opened in a new tab and note that the tab count is incremented and animates.
1. Check that the original tab is unchanged and still on the w3cschools website.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
